source/iface/param_traits.rst

```diff
@@ -2,23 +2,197 @@
   Copyright 2020 The Khronos Group Inc.
   SPDX-License-Identifier: CC-BY-4.0
 
-================
-``param_traits``
-================
+************************
+Backend interoperability
+************************
+
+Many of the SYCL runtime classes may be implemented such that they
+encapsulate an object unique to the SYCL backend that underpins the
+functionality of that class. Where appropriate, these classes may
+provide an interface for interoperating between the SYCL runtime
+object and the native backend object in order to support
+interoperability within an application between SYCL and the
+associated SYCL backend API.
+
+There are three forms of interoperability with SYCL runtime classes:
+interoperability on the SYCL application with the SYCL backend API,
+interoperability within a SYCL kernel function with the equivalent
+kernel language types of the SYCL backend, and interoperability
+within a host task with the ``interop_handle``.
+
+SYCL application interoperability, SYCL kernel function interoperability
+and host task interoperability are provided via different interfaces
+and may have different behavior for the same SYCL object.
+
+SYCL application interoperability may be provided for ``buffer``, ``context``,
+``device``, ``device_image``, ``event``,`` kernel``, ``kernel_bundle``,
+``platform``, ``queue``, ``sampled_image``, and ``unsampled_image``.
+
+SYCL kernel function interoperability may be provided for ``accessor``,
+``device_event``, ``local_accessor``, ``sampled_image_accessor``, ``tream``
+and ``unsampled_image_accessor`` inside kernel scope only and
+is not available outside of that scope.
+
+host task interoperability may be provided for ``accessor``,
+``sampled_image_accessor``, ``unsampled_image_accessor``, ``queue``,
+``device``, ``context`` inside the scope of a host task only.
+
+.. seealso:: |SYCL_SPEC_HOST_TASK|
```

Comment:
I suggest instead linking to 4.5.1 which is more generally about interoperability, instead of only about host tasks.

Reply:
Fixed wit commit : https://github.com/KhronosGroup/SYCL_Reference/pull/93/commits/883a39dc43de3f37be6d2c71d991b079a93f1ba0